lib/sla.js

```diff
@@ -84,12 +123,38 @@ function createChannel(client, bridge) {
  * @param {Object} client - Client received from app.js.
  * @return {Q} Q promise object.
  */
-module.exports = function(client) {
-  return designateMixingBridge(client)
-    .then(function (bridge) {
-      return createChannel(client, bridge)
-        .then(function (channel) {
-          return addChannelsToBridge(channel, bridge);
-        });
-    });
+module.exports = function(client, channel, bridgeNumber) {
+  //This specifies what bridge number to use for SLA
+  if(!isNaN(parseInt(bridgeNumber))) {
+    var answer = Q.denodeify(channel.answer.bind(channel));
+    answer();
```

Comment:
This may or may not be done running before you run the next ARI request since it's not being used with a .then clause.